osclib/conf.py

```diff
@@ -161,7 +119,6 @@
         'lock-ns': None,
         'delreq-review': None,
         'main-repo': 'openSUSE_Factory',
-        'remote-config': False,
```

Comment:
This will break the gnome review bot unless either `attribute_value_load()` is changed to handled `404`.